### PR TITLE
Fix decode_stream.py

### PR DESCRIPTION
**Pull request by yfyeung**

### CODE REVIEW

File: egs/librispeech/ASR/zipformer/decode_stream.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def __init__(
         self.pad_length = 7 + 2 * 3
 
         if params.decoding_method == "greedy_search":
-            self.hyp = [params.blank_id] * params.context_size
+            self.hyp = [-1] * (params.context_size - 1) + [params.blank_id]
```
Comment (csukuangfj) [inline egs/librispeech/ASR/zipformer/decode_stream.py:82]: Please also update Line 87

**Review by csukuangfj — APPROVED** [review]

**Comment by ezerhouni** [comment]
@csukuangfj I think the changes has not been made for modified_beam_search. Should I open a PR to fix it ?

**Comment by csukuangfj** [comment]
> @csukuangfj I think the changes has not been made for modified_beam_search. Should I open a PR to fix it ?

Yes, please. Thanks!

**Comment by ezerhouni** [comment]
@csukuangfj Great, will do by tomorrow

**Pull request merged**